docs/proposals/perf.md

```diff
@@ -298,6 +298,29 @@ Test Cases:
 
 These test cases should be run in both system idle and under heavy load.
 
+#### 6. Add Pod from CloudHub to EdgeHub
+<img src="../images/perf/perf-cloudhub-edgehub.png">
+
+This scenario is expected to measure the performance of KubeEdge between CloudHub to EdgeHub.
+Actually this is not an E2E Test scenario for KubeEdge,
+but the message delivery channel between CloudHub to EdgeHub may be our bottleneck.
+Currently we are using web socket as the communication protocal between Cloud and Edge.
+In the following test cases, we need to mock the behaviors of CloudHub and EdgeHub,
+and the simulation messages of adding pod will be sent to EdgeHub,
+and the simulation messages of pod status will be sent back to CloudHub.
+so that we can get the exact latency and throughput between CloudHub and EdgeHub.
+
+Test Cases:
+* Measure the latency between KubeEdge CloudHub and KubeEdge EdgeHub.
```

Comment:
Please pay good attention on
-  when bunch of edge nodes got disconnected & then reconnect. 
-  cloudhub CPU/memory when there are large scale of edge nodes where cloudhub is sending/receiving messages.

Reply:
Thanks @CindyXing . The CPU/memory is added in this scenario.
we will consider disconnected & then reconnect case once we have the original test data for this scenario.